cache/solidity-files-cache.json

```diff
@@ -0,0 +1,1548 @@
+{
```

Comment:
I think this file can be removed from the commit

Reply:
Removed and updated .gitignore